airbyte-commons-server/src/test/java/io/airbyte/commons/server/handlers/ConnectorBuilderProjectsHandlerTest.java

```diff
@@ -97,6 +108,26 @@ void testUpdateConnectorBuilderProject() throws IOException, JsonValidationExcep
             project);
   }
 
+  @Test
+  @DisplayName("updateConnectorBuilderProject should update an existing project removing the draft")
+  void testUpdateConnectorBuilderProjectWipeDraft() throws IOException, ConfigNotFoundException {
+    final ConnectorBuilderProject project = generateBuilderProject();
+    project.setManifestDraft(null);
```

Comment:
is the test's DisplayName right? It looks like the manifest is set to null before running the update

Reply:
It is correct, the order of operations is a little confusing - it is set to `null` for the validation by the end of the file whether the project passed to `writeBuilderProject` doesn't have a draft. I rearranged the code to make this clearer.